public/js/index.js

```diff
@@ -362,6 +363,17 @@ async function removeLesson(deleteId) {
 	}
 }
 
+modalLessonCloseByESC.addEventListener('keydown', event => {
```

Comment:
The modal still doesn't close when I hit the Esc key. Instead of attaching the event listener to the close button, we should probably attach this to the `document`, otherwise the keydown event won't be captured.

Let me know if this doesn't make sense 😄